setup.py

```diff
@@ -17,11 +17,11 @@
         "dill",
         "matplotlib",
         "requests",
-        "numpy==1.19.5",
+        "numpy",
         "seaborn==0.9.0",
         "pygame==1.9.5",
-        "ray[rllib]==0.8.5",
+        "ray[rllib]==2.0.0",
```

Comment:
Do we need `==` here? Could we have `>=`? If I remember correctly, this is what you were suggesting as being best practice in these kinds of cases (having the least-stringent requirements as possible)